cornflow/commands/commands.py

```diff
@@ -55,36 +69,59 @@ def get_options(self):
         return (
             Option("-e", "--email", dest="email", help="Service user email"),
             Option("-p", "--password", dest="password", help="Service user password"),
+            Option(
+                "-v",
+                "--verbose",
+                dest="verbose",
+                help="Verbose for the command. 0 no verbose, 1 full verbose",
+            ),
         )
 
-    def run(self, email, password):
+    def run(self, email=None, password=None, verbose=0):
         """
         Method to run the command and create the service user
 
         :param str email: the email for the admin user
         :param str password: the password for the admin user
+        :param int verbose: verboseof the command
         :return: a boolean if the execution went right
         :rtype: bool
         """
-        return create_user_with_role(email, password, "serviceuser", SERVICE_ROLE)
+        if email is None or password is None:
+            print("Missing required arguments")
+        verbose = int(verbose)
+        return create_user_with_role(
+            email, password, "serviceuser", SERVICE_ROLE, verbose
+        )
 
 
 class CreateAdminUser(Command):
     def get_options(self):
         return (
             Option("-e", "--email", dest="email", help="Admin user email"),
             Option("-p", "--password", dest="password", help="Admin user password"),
+            Option(
+                "-v",
+                "--verbose",
+                dest="verbose",
+                help="Verbose for the command. 0 no verbose, 1 full verbose",
+            ),
         )
 
-    def run(self, email, password):
+    def run(self, email=None, password=None, verbose=0):
         """
         Method to run and create the admin user
+
         :param str email: the email for the admin user
         :param str password: the password for the admin user
+        :param int verbose: verbose of the command
         :return: a boolean if the execution went right
         :rtype: bool
         """
-        return create_user_with_role(email, password, "admin", ADMIN_ROLE)
+        if email is None or password is None:
+            print("Missing required arguments")
+        verbose = int(verbose)
```

Comment:
you can pass a type argument to the Option I think so you don't have to cast to int: https://flask-script.readthedocs.io/en/latest/#flask_script.Option

Reply:
Ok, will modify it